setup.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+
+read -p "Enter the site name: " site_name
+read -p "Enter the theme name: " theme_name
+
+
+BASE_DIR="docroot/sites/$site_name/themes/custom/$theme_name"
```

Comment:
@sandipta18 We are hardcoding `docroot` here but some sites use `web`, `webroot`, etc.

Reply:
Updated the script to fetch base directory information from user